bootstrap.sh

```diff
@@ -1,39 +1,505 @@
 #!/bin/bash
+#
+# *******************************************************************************
+#  * Copyright (c) 2019 Edgeworx, Inc.
+#  *
+#  * This program and the accompanying materials are made available under the
+#  * terms of the Eclipse Public License v. 2.0 which is available at
+#  * http://www.eclipse.org/legal/epl-2.0
+#  *
+#  * SPDX-License-Identifier: EPL-2.0
+#  *******************************************************************************
+#
+
+. ./scripts/utils.sh
 
 OS=$(uname -s | tr A-Z a-z)
-GCLOUD_VERSION=240.0.0
+GCLOUD_VERSION=253.0.0
+GCP_SDK_PACKAGE_URL="https://console.cloud.google.com/storage/browser/cloud-sdk-release?authuser=0&prefix=google-cloud-sdk-${GCLOUD_VERSION}"
+GCP_SDK_INSTRUCTION_URL="https://cloud.google.com/sdk/docs/downloads-versioned-archives"
+LIB_LOCATION=/usr/local/lib
+
 TERRAFORM_VERSION=0.11.14
+TERRAFORM_PACKAGE_URL="https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/"
+TERRAFORM_INSTRUCTION_URL="https://cloud.google.com/sdk/docs/downloads-versioned-archives"
+
+KUBECTL_INSTRUCTION_URL="https://kubernetes.io/docs/tasks/tools/install-kubectl/"
+
+ANSIBLE_INSTRUCTION_URL="https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#intro-installation-guide"
+
+help_install_gcp_sdk() {
+    echoError "We could not automatically install gcloud sdk"
+    echoInfo "Please follow the installation instructions from here: ${GCP_SDK_INSTRUCTION_URL}"
+}
 
-# GCP CLI
-if [[ -z $(command -v gcloud) ]]; then
-    if [ "$OS" = "darwin" ]; then
-        curl -Lo gcloud.tar.gz https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-"$GCLOUD_VERSION"-"$OS"-x86_64.tar.gz
-        tar -xf gcloud.tar.gz
-        rm gcloud.tar.gz
-        google-cloud-sdk/install.sh -q
-        rm -r google-cloud-sdk
+install_gcp() {
```

Comment:
not sure this is taken care of somewhre else, but after installation, we have to `gcloud init` or auth as something before using it